packages/ui/src/primitives/tailwind/Checkbox/index.tsx

```diff
@@ -48,7 +50,26 @@ const Checkbox = ({ className, containerClassName, label, value, onChange, disab
         }
       }}
       className={twMerge('flex cursor-pointer items-end space-x-2', containerClassName)}
+      ref={ref}
+      onKeyDown={(event) => {
+        if (event.key === 'Enter') {
+          if (!disabled) {
+            onChange(!value)
+          }
+        }
+      }}
     >
+      <input
+        type="checkbox"
+        className="scale-x-0 scale-y-0"
+        onFocus={(e) => {
+          ref.current?.focus()
+          ref.current?.classList.add('border-2', 'border-white')
```

Comment:
just need to use this color for border - <https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2786-21102&t=KskN2TYOIK1veI31-0>